service/src/main/java/bio/terra/catalog/service/DatasetService.java

```diff
@@ -229,6 +234,18 @@ public DatasetPreviewTablesResponse listDatasetPreviewTables(
     return new DatasetPreviewTablesResponse().tables(tableMetadataList);
   }
 
+  private void validateSchema(String metaData) {
```

Comment:
To be consistent with other code in our repo, this should be spelled `metadata`
```suggestion
  private void validateSchema(String metadata) {
```